inspector/src/components/actionTabs/tabs/propertyGrids/animations/curveEditor.scss

```diff
@@ -660,7 +661,7 @@ body {
         }
 
         .timeline-wrapper {
-          margin-top: -45px;
+          margin-top: -31px;

```

Comment:
Where is this value coming from? 31px?

Reply:
This change is just to allow a better alignment here 
![image](https://user-images.githubusercontent.com/704682/88677973-8eb26200-d0b3-11ea-99ee-df9fa8905585.png)